features/upgrade/sdn/ipsec-upgrade.feature

```diff
@@ -0,0 +1,93 @@
+Feature: IPsec upgrade scenarios
+
+  # @author anusaxen@redhat.com
+  @admin
+  @upgrade-prepare
+  Scenario: Confirm node-node and pod-pod packets are ESP enrypted on IPsec clusters post upgrade
+    Given the env is using "OVNKubernetes" networkType
+    And the IPsec is enabled on the cluster
+    Given I store all worker nodes to the :workers clipboard
+    Given I switch to cluster admin pseudo user
+    When I run the :new_project client command with:
+      | project_name | ipsec-upgrade |
+    Then the step should succeed
+    When I use the "ipsec-upgrade" project
+    Given I obtain test data file "networking/list_for_pods.json"
```

Comment:
@anuragthehatter You may wan to confirm the image used in the JSON is mirrored when the cluster is disconnected.

Reply:
its our basic hello-sdn image so thats already mirrored